README.md

```diff
@@ -101,6 +101,7 @@ All the possible environment variables are [defined here](config/custom-environm
 This API comes preloaded with 9 (nine) resources:
```

Comment:
should be `10 (ten)`
Also need to update branch to be consistent with master
I think it's already `10 resources` currently on master